docs/source/tutorials/oregon.rst

```diff
@@ -0,0 +1,688 @@
+             title=f"{stratum}\n(Simple Local Estimator, n={results['sample_size']:,})",
+             xlabel="Emergency Department Costs", ylabel="Local Distribution Treatment Effect",
+             color="blue" if col_idx == 1 else "green", ax=axes[0, col_idx])
+        col_idx += 1
+
+    # Row 2: ML-Adjusted local estimators
+    # Overall (all data)
+    plot(outcome_ed_costs_locations, ldte_ml, lower_ml, upper_ml,
+         title="Overall Population\n(ML-Adjusted Local Estimator)",
+         xlabel="Emergency Department Costs", ylabel="Local Distribution Treatment Effect",
+         color="black", ax=axes[1, 0])
+
+    # Individual strata
+    col_idx = 1
+    for stratum, results in individual_results.items():
+        if results is None or col_idx > 2:
+            continue
+
+        plot(outcome_ed_costs_locations, results['ml']['ldte'],
+             results['ml']['lower'], results['ml']['upper'],
+             title=f"{stratum}\n(ML-Adjusted Local Estimator, n={results['sample_size']:,})",
+             xlabel="Emergency Department Costs", ylabel="Local Distribution Treatment Effect",
+             color="red" if col_idx == 1 else "purple", ax=axes[1, col_idx])
+        col_idx += 1
+
+    plt.suptitle("Comparison: Overall Population vs Individual Household Registration Strata (Local Estimators)", fontsize=16)
+    plt.tight_layout()
+    plt.show()
+
+.. image:: ../_static/oregon_ldte_strata.png
+   :alt: Oregon Health Insurance Experiment DTE Strata Analysis
+   :width: 800px
+   :align: center
+
+**Key Insights from Stratified Analysis**:
+
+The stratified analysis by household registration type reveals important heterogeneity in how Medicaid assignment affects different populations:
+
+**Heterogeneous Local Treatment Effects**: The comparison between overall population effects and individual strata shows that local treatment effects vary significantly across different household registration patterns. This heterogeneity suggests that "one-size-fits-all" policy evaluations may miss important subgroup differences in complier populations.
+
+**Sample Size Considerations**: Different strata have varying sample sizes, which affects the precision of estimates. Larger strata (like "signed self up") provide more precise estimates, while smaller strata show wider confidence intervals but may reveal important effect heterogeneity among compliers.
+
+**Policy Targeting Implications**: Understanding which household types respond most strongly to Medicaid assignment can inform more targeted policy interventions and help identify populations that would benefit most from expanded coverage when they actually enroll.
+
+**Methodological Consistency**: Both simple and ML-adjusted local estimators show similar patterns within each stratum, providing confidence in the robustness of the stratified findings across different analytical approaches while accounting for non-compliance.
+
+Conclusion
+~~~~~~~~~~
```

Comment:
Again, can we ground the insights on the Oregon analysis results? The current summary feels too generic.